packages/x-components/src/views/home/sliding-query-preview.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <QueryPreview :query="query" #default="{ results, totalResults, query }">
+    <div class="x-list x-list--gap-03 x-margin--bottom-05">
+      <h1 class="x-title2 x-text--bold">{{ query }} ({{ totalResults }})</h1>
```

Comment:
Also, in this task: https://searchbroker.atlassian.net/browse/EX-6925

We are describing that will need to receive a title and a query in the snippet. Do we need to add a prop in the title, or will we add the prop on a custom wrapper component in the Archetype?

Reply:
We don't need to add it here. This `SlidingQueryPreview` is used just for the demo of `x-components`. Later in the Archetype, we will use the title provided by the snippet config info in a custom wrapper component, as you said.